delivery_recipe/kick_starting_a_project.md

```diff
@@ -0,0 +1,64 @@
+# Kick Starting a Project
+
+Once a project has been sold the next step is usually an intense discovery process to map out processes, to look at possible solutions and to identify the high level requirements (epics). At the end of the discovery process the next step is to hand the client over to the project delivery team to begin the process of developing what’s been agreed.
+
+## Where to start
+
+It can be daunting when you first begin a project. The expectations are high, there is excitement among those who have been involved in the process so far and it’s your job to pick up the baton and see this through to the finish.
+
+The first step in the process of taking the project into the delivery phase is the handover from the discovery or sales team to the project delivery team. To maintain continuity and the trust of the client the handover will involve:
```

Comment:
Actually I see this as both. The delivery team effectively take on the project from the sales/discovery team so it is a handover. I expect that this would be an extended first meeting that would roll into the first planning session.